issueWork/HTML__issue/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script defer>
+      document.querySelector();
```

Comment:
This is unneeded for now

Reply:
Oops I forgot it was in there. After I finished it I was just going to make this document meet all the requirement for both issues 3 && 4 and give the form submit a funny little browser alert. In the end I decided not to last night and went to dinner with my wife. I just forgot to remove it. Sorry!

Reply:
No need to apologize, ur good